<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteEditorWidget</class>
 <widget class="QWidget" name="PaletteEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>740</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Palette Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="intepolationButton">
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Toggle palette value interpolation</string>
       </property>
       <property name="text">
        <string>I</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addToCollectionButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="importButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PaletteTableView" name="tableViewPalette">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonAddCtrlPoint">
       <property name="toolTip">
        <string>Add new control point</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDelCtrlPoint">
       <property name="toolTip">
        <string>Delete selected control point</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaletteTableView</class>
   <extends>QTableView</extends>
   <header>palettetableview.h</header>
   <slots>
    <slot>addControlPoint()</slot>
    <slot>delControlPoint()</slot>
    <slot>toggleInterpolation(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonAddCtrlPoint</sender>
   <signal>released()</signal>
   <receiver>tableViewPalette</receiver>
   <slot>addControlPoint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>718</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDelCtrlPoint</sender>
   <signal>released()</signal>
   <receiver>tableViewPalette</receiver>
   <slot>delControlPoint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>718</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intepolationButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>tableViewPalette</receiver>
   <slot>toggleInterpolation(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>718</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
